knowledge/miscellaneous_unknown/qna.yaml

```diff
@@ -0,0 +1,64 @@
+created_by: gtyellow
+version: 3
+domain: Sports
+document_outline: >-
+  These are the international rules for soccer.  They are called the laws of the
+  game.
+seed_examples:
+  - context: This is the rules of the shape and dimensions of a field
```

Comment:
The `context` should be a specific section from the referenced document that you declare at the bottom of this `yaml`.

Reply:
RE: https://github.com/instructlab/taxonomy/blob/main/README.md#knowledge-yaml-examples